packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -1151,26 +1151,39 @@ describe('@api RPC Server Acceptance Tests', function () {
 
         describe('RPC rate limit test', () => {
             it('should throw rate limit exceeded error', async function() {
+                let rateLimited = false;
                 try{
```

Comment:
Does a code formatter need to be run against this?  `try {` not `try{`

Reply:
Yeah. We should get prettier running on commit or something.